packages/js/api/src/models/products/abstract/inventory.ts

```diff
@@ -56,6 +56,13 @@ abstract class AbstractProductInventory extends Model {
 	 * @type {boolean}
 	 */
 	public readonly isOnBackorder: boolean = false;
+
+	/**
+	 * Indicates when the threshold for when low stock notification will be sent to the store admin.
+	 *
+	 * @type {boolean}
+	 */
+	public readonly lowStockThreshold: number = -1;
```

Comment:
The `@type` annotation here indicates this is a `boolean` however the data type assigned here is `number`. Should the type be `number` instead of `boolean` here?

Reply:
Thanks for catching this! It should be `boolean`. I've made the update